ecs-init/config/common.go

```diff
@@ -124,6 +124,11 @@ func DockerUnixSocket() (string, bool) {
 	return "/var/run", false
 }
 
+// CgroupMountpoint returns either the default cgroup mountpoint for the system or the mountpoint overidden at CGROUP_MOUNTPOINT
```

Comment:
I think this should be updated now that we aren't going to have env override support.

Reply:
Agreed!